[2022-08-16 18:35:11.771] [remoteagent] [info] Extension host agent started.
[2022-08-16 18:35:12.481] [remoteagent] [info] [127.0.0.1][a2f30a12][ManagementConnection] New connection established.
[2022-08-16 18:35:12.482] [remoteagent] [info] [127.0.0.1][60f92a32][ExtensionHostConnection] New connection established.
[2022-08-16 18:35:12.489] [remoteagent] [info] [127.0.0.1][60f92a32][ExtensionHostConnection] <24738> Launched Extension Host Process.
[2022-08-16 18:35:27.866] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:24:29654)","    at d.updateIcon (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:24:25490)","    at Object.call (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:16:8327)","    at a.onPromise (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:15:4892)","    at a.onRawMessage (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:15:4275)","    at /home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:15:3561","    at f.invoke (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at N.deliver (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at I.fire (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:9:20673)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-08-16 18:35:27.872] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:24:29654)","    at d.updateTitle (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:24:25430)","    at Object.call (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:16:8327)","    at a.onPromise (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:15:4892)","    at a.onRawMessage (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:15:4275)","    at /home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:15:3561","    at f.invoke (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at N.deliver (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at I.fire (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/platform/terminal/node/ptyHostMain.js:9:20673)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-08-16 18:37:52.562] [remoteagent] [info] [127.0.0.1][60f92a32][ExtensionHostConnection] The client has reconnected.
[2022-08-16 18:38:19.471] [remoteagent] [info] [127.0.0.1][a2f30a12][ManagementConnection] The client has reconnected.
[2022-08-16 18:38:19.647] [remoteagent] [info] [127.0.0.1][60f92a32][ExtensionHostConnection] The client has reconnected.
[2022-08-16 18:46:44.860] [remoteagent] [info] [127.0.0.1][60f92a32][ExtensionHostConnection] The client has reconnected.
[2022-08-16 19:17:11.807] [remoteagent] [info] [127.0.0.1][a2f30a12][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-08-16 19:17:11.807] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/miura/.vscode-server/bin/6d9b74a70ca9c7733b29f0456fd8195364076dda/out/vs/server/node/server.main.js:145:7922)
    at process.emit (node:events:390:28)
[2022-08-16 21:46:58.504] [remoteagent] [info] [127.0.0.1][60f92a32][ExtensionHostConnection] <24738> Extension Host Process exited with code: 0, signal: null.
[2022-08-16 21:46:58.504] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-08-16 21:51:58.506] [remoteagent] [info] Last EH closed, shutting down
